packages/nimble-components/src/table-column/number-text/tests/table-column-number-text.stories.ts

```diff
@@ -105,6 +106,21 @@ const formatDescription = `Configures the way that the numeric value is formatte
 </details>
 `;
 
+const alignmentDescription = `Configures the alignment of the value within the column.
+
+<details>
+    <summary>Default Alignment</summary>
+
+    The default alignment of the value depends on the column's format.
+    <ul>
+        <li>\`default\` format: Values are left-aligned by default.
+        </li>
+        <li>\`roundToInteger\` format: Values are right-aligned by default.
```

Comment:
I think it's clearer without "by default".
```suggestion
        <li>\`default\` format: Values are left-aligned.
        </li>
        <li>\`roundToInteger\` format: Values are right-aligned.
```